### PR TITLE
Added a task Bob

### PR DESCRIPTION
**Pull request by annaoliinyk**
https://exercism.org/tracks/python/exercises/bob

Instructions
Bob is a lackadaisical teenager. In conversation, his responses are very limited.

Bob answers 'Sure.' if you ask him a question, such as "How are you?".

He answers 'Whoa, chill out!' if you YELL AT HIM (in all capitals).

He answers 'Calm down, I know what I'm doing!' if you yell a question at him.

He says 'Fine. Be that way!' if you address him without actually saying anything.

He answers 'Whatever.' to anything else.

Bob's conversational partner is a purist when it comes to written communication and always follows normal rules regarding sentence punctuation in English.

### CODE REVIEW

File: bob/bob_task.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        if hey_bob != hey_bob.upper():
+            return 'Sure.'
+        elif hey_bob != hey_bob.lower():
+            return 'Calm down, I know what I\'m doing!'
+        else:
+            return 'Sure.'
+
+    elif hey_bob != hey_bob.lower() and hey_bob == hey_bob.upper():
+        return 'Whoa, chill out!'
+
```
Comment (AnaKovaGH) [inline bob/bob_task.py:37]: Unnecessary space

File: bob/bob_task.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Task from https://exercism.org/tracks/python/exercises/bob
+
+import re
+
+
+def response(hey_bob):
+    if hey_bob.strip() == "":
+        return 'Fine. Be that way!'
```
Comment (AnaKovaGH) [inline bob/bob_task.py]: Please choose one rule regarding quotation marks and use it everywhere in the code

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please change what was requested above

**Comment by annaoliinyk** [comment]
This was fixed

**Comment by annaoliinyk** [comment]
Ready for review

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please remove hard-coded values in the return statements (e.g. return POSSIBLE_ANSWERS[2]), use dictionary instead.
This this bad approach, because in case if you change the order or add new answers to the list, the logic will be broken.

**Comment by annaoliinyk** [comment]
Done

**Review by AnaKovaGH — APPROVED** [review]

**Pull request merged**